seabird_chat_ingest.proto

```diff
@@ -0,0 +1,144 @@
+syntax = "proto3";
+
+option go_package = ".;pb";
+
+import "common.proto";
+
+package seabird;
+
+// Note that all of the types in this file end with ChatRequest or end with
+// ChatEvent to prevent namespace conflicts between the main seabird.proto file.
```

Comment:
Why is this necessary? Protobuf imports are already namespaced, and you control naming on code import.

Reply:
With go when you build all these protos into the same go package, they can conflict.

Reply:
That begs the question though: why are these the same Go package?

Reply:
Common practice is to put generated protobufs in a `pb` package, even when coming from multiple protobufs. For some reason when you `import` another protobuf in Go, it imports it into the same namespace, so those common.User messages are still just exported as User.

Reply:
Punting on this for now